index.d.ts

```diff
@@ -2,4 +2,4 @@
 
 
 
-/// <reference path="./node_modules/@types/lodash/index.d.ts" />
\ No newline at end of file
+/// <reference path="./node_modules/@types/lodash/index.d.ts" />
```

Comment:
what changed here?
So we dont need to add the http-mock thing?
Are we even using it? .. did you try uninstalling the types and see if it makes a difference?